WhateverGreen/kern_rad.cpp

```diff
@@ -302,6 +307,21 @@ IOReturn RAD::wrapAMDRadeonX6000AmdRadeonFramebufferGetAttribute(IOService *fram
 	return ret;
 }
 
+bool RAD::ifNeedOverrideConnector(KernelPatcher &patcher, size_t index, mach_vm_address_t address, size_t size) {
+	// No need to overwrite the connector when booting BaseSystem on macOS Tahoe.
+	// This step is unnecessary and will cause macOS Tahoe recovery mode to freeze when booting.
+	// Committed by laobamac
+	if (getKernelVersion() >= KernelVersion::Tahoe) {
+		if (kextRadeonServiceManager.loadIndex != index) {
```

Comment:
ifNeedOverrideConnector is only ever called when index == kextRadeonSupport.loadIndex. Does this not have the affect of removing all connector patches in all Tahoe versions?